SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -115,12 +156,16 @@ public static void Main(string[] args)
                     Directory.CreateDirectory(windowsEditorPath);
                 }
 
-                // Write a simple batch file to launch the Editor as a managed worker.
-                File.WriteAllText(Path.Combine(windowsEditorPath, "StartEditor.bat"),
+                string RunEditorScript = @"setlocal ENABLEDELAYEDEXPANSION" + "\n" + unrealEngine + @"\Engine\Binaries\Win64\UE4Editor.exe ""{0}"" %*
```

Comment:
Use string interpolation for readability:
```
var RunEditorScript = $@"setlocal ENABLEDELAYEDEXPANSION
{Path.Combine(unrealEngine, "Engine\Binaries\Win64\UE4Editor.exe")} ""{0}"" %*
exit /b !ERRORLEVEL!
";```

Reply:
Took a bit of tweaking and learning but got it going. Clean :D